### PR TITLE
MAT-7679 date time component

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-7679](https://jira.cms.gov/browse/MAT-7679)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [x] Required tests are included
* [x] No extemporaneous files are included (i.e Complied files or testing results)
* [x] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/types/DateTimeComponent.tsx [hunk 1]
```diff
@@ -1,16 +1,233 @@
...
+  if (value && options) {
+    options.forEach((opt) => {
+      if (opt.label === value) {
+        result = opt.value + " - " + opt.label;
+      }
+    });
+  }
+  return result;
+};
+const convertToStandartTime = (dayLightTime) => {
```
Comment (jkotanchik-SB) [inline src/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/types/DateTimeComponent.tsx]:
```suggestion
const convertToStandardTime = (dayLightTime) => {
```

Comment (nmorasb) [reply src/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/types/DateTimeComponent.tsx]: not sure why this was marked resolved as it seems like the name still should be updated?

Comment (sb-cecilialiu) [reply src/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/types/DateTimeComponent.tsx]: @nmorasb Just fixed the typo, thanks!

File: src/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/types/DateTimeComponent.tsx [hunk 2]
```diff
@@ -1,16 +1,233 @@
...
+  { value: "America/Los_Angeles", label: "PST" },
+];
+
+const offSets = [
+  { id: "America/New_York - EST", value: "-05:00" },
+  { id: "America/Chicago - CST", value: "-06:00" },
+  { id: "America/Denver - MST", value: "-07:00" },
+  { id: "America/Los_Angeles - PST", value: "-08:00" },
+];
+
```
Comment (nmorasb) [inline src/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/types/DateTimeComponent.tsx:40]: Were these timezones and offsets confirmed with POs? Are we sure we should limit only to these four? (Even if only limited to US, what about Hawaii, Alaska, etc?)

Comment (sb-cecilialiu) [reply src/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/types/DateTimeComponent.tsx:40]: @nmorasb Thanks Nate! I posted the question to POs in the jira card.

Comment (nmorasb) [reply src/components/editTestCase/qiCore/LeftPanel/ElementsTab/builder/element/types/DateTimeComponent.tsx:40]: approving for now, but let's keep this jira in code review until we have an answer on that. At least then if any change are needed, the subsequent PR will be more limited

**Review by nmorasb — APPROVED** [review]
